projects/yfdai/index.js

```diff
@@ -0,0 +1,42 @@
+const Web3 = require('web3');
+require("dotenv").config();
+let web3RpcUrl = process.env.ETHEREUM_RPC;
+const web3 = new Web3(new Web3.providers.HttpProvider(web3RpcUrl));
+module.exports = web3;
+const retry = require('../helper/retry')
+const axios = require("axios");
+const BigNumber = require("bignumber.js");
+const abi = require('./abi');
+
+async function fetch() {
+
+    const yfdaiTokenAddress = "0xf4CD3d3Fda8d7Fd6C5a500203e38640A70Bf9577";
+
+    const YfDaiETHLP72HRSAddress = "0x75E9F410e8d1D7240b67ec6FE35FA37580b814d9";
+    const YfDaiETHLP30DayAddress = "0x8D704D4107CBE5ebE8c0236C5506b30Bf8Bad305";
+    const YfDaiETHLP60DayAddress = "0x26572bf2620108cb5006987e6348c07dc4e14a0f";
+    const YfDaiETHLP90DayAddress = "0x175d6cbaeff93734ada4c5430815f2208a6b040c";
+    const YfDaiStakingAdddress = "0x44d771D0C998f524ff39aB6Df64B72bce1d09566";
+    const YfDaiSafetradeStakingAddress = "0x4599cDa238Fb71573fd5A0076C199320e09BCfF0";
```

Comment:
Could you explain what is this contract?

Reply:
YfDaiETHLP60DayAddress:  Farm for Uniswap LP tokens(with 60 days locking)
YfDaiETHLP90DayAddress: Farm for Uniswap LP tokens (with 90 days locking)
YfDaiStakingAdddress: Staking contract(Users can stake YFDAI and get YFDAI as rewards)
YfDaiSafetradeStakingAddress:  SafeTrade is a bot where users can deposit YFDAI and the Bot can use this to trade in markets.

Reply:
Given that the amount of tokens in staking heavily dwarfs the amount of tokens on safeTrading I believe it would be best to exclude staking for now, since no other platform we have on defillama has this much staking

Reply:
Right now,
Staking has ~11816 YFDAI tokens 
SafeTrade has ~ 157 YFDAI tokens
So you mean You won't be counting Staking tokens for TVL?
It should be counted along with all the farms right?

Reply:
Yes, I believe it's best not to count staking to keep in line with what we count in other projects

Reply:
But, 11816 YFDAs are a major chunk of our supply, and it's locked, so it should be counted in TVL.
If there is some technical challenge right now, then let's go ahead and list YFDAI for now, and then you can figure out about staking tokens later.

Reply:
When can we expect this pull request to be merged?

Reply:
Yes, I went ahead and wrote an alternative version of the adapter on https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/yfdai/index.js, let me know what you think

Reply:
I think it's good to go. Please merge it to master/main.

Reply:
@0xngmi Even after the YFDAI Project is merged in main, we are not able to see the token in the application.